test/helpers/generateToken.test.js

```diff
@@ -0,0 +1,28 @@
+import chai from 'chai';
+import helpers from '../../server/helpers';
```

Comment:
please update your test to show that when an expiresIn is not provided, we use a default value